### PR TITLE
Some minor improvements

### PR DESCRIPTION
**Pull request by finngaida**
- Start the animation with an empty set
- Faster animations
- Bigger Apple
- Readme

### CODE REVIEW

File: WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift [hunk 1]
```diff
@@ -197,23 +197,20 @@ class WWDC2016ScreenSaverView: ScreenSaverView {
     }
     
     func tick() {
-        if self.isAnimating == false {
-            return
-        }
         
-        if textLayers.count == 0 {
-            return
+        func restart() {
```
Comment (justMaku) [inline WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift:201]: Any particular reason you don't want to use CAAnimation?

File: WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift [hunk 2]
```diff
@@ -197,23 +197,20 @@ class WWDC2016ScreenSaverView: ScreenSaverView {
...
+            DispatchQueue.main.after(when: DispatchTime.now() + 0.05) {
+                self.tick()
+            }
         }
         
-        CATransaction.setCompletionBlock {
-            self.tick()
+        if self.isAnimating == false {
+            restart()
```
Comment (justMaku) [inline WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift]: No need to restart if it's not animating.

File: WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift [hunk 3]
```diff
@@ -38,7 +38,7 @@ class WWDC2016ScreenSaverView: ScreenSaverView {
     let words = [":", ";", "\\", "/", ".", "!", "?",
                  "+", "-", "*", "&", "^", "[", "]",
                  "(", ")", "#", "@", "&", "<", ">",
-                 "~"]
+                 "~", " "/*, "do", "let", "var", "try", "if", "for", "as"*/]
```
Comment (justMaku) [inline WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift]: Let's remove the comment for now.

File: WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift [hunk 4]
```diff
@@ -197,23 +197,20 @@ class WWDC2016ScreenSaverView: ScreenSaverView {
...
+        if self.isAnimating == false {
+            restart()
         }
-
         
-        CATransaction.begin()
-        CATransaction.setAnimationDuration(1.0)
-        CATransaction.setValue(kCFBooleanTrue, forKey: kCATransactionDisableActions)
-        CATransaction.setAnimationTimingFunction(CAMediaTimingFunction(name: kCAMediaTimingFunctionLinear))
+        if textLayers.count == 0 {
```
Comment (justMaku) [inline WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift]: Again, I'm not sure we should be ticking if there's nothing to animate.

File: WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift [hunk 5]
```diff
@@ -119,7 +119,7 @@ class WWDC2016ScreenSaverView: ScreenSaverView {
             if percentage >= 1 {
                 let drawingPoint = CGPoint(x: point.x + newOrigin.x, y: point.y + newOrigin.y)
                 let textLayer = CATextLayer()
-                textLayer.string = word
+                textLayer.string = " " //word
```
Comment (justMaku) [inline WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift]: Could we remove the coment please?

**Comment by justMaku** [comment]
I love the idea with starting from empty screen! <3 Thanks a lot for this PR, some minor adjustments are needed but 👌 otherwise.

File: WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift (hunk 1 above)
Comment (finngaida) [inline WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift:201]: I found it to be lagging in the screensaver for some reason, so I took it out to test, but forgot to put it back in, would probably be better.

File: WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift (hunk 3 above)
Comment (finngaida) [inline WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift]: It'd be cool to have those multi char words in there as well, but there would be more logic necessary, so I didn't yet implement that

File: WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift (hunk 5 above)
Comment (finngaida) [inline WWDC 2016 ScreenSaver/WWDC2016ScreenSaver.swift]: haha, sure

**Comment by finngaida** [comment]
I'll push the adjustments soon, sure, thanks for open sourcing!

**Comment by justMaku** [comment]
Just one more thing: could you please remove the references to the reference movie as we won't be using it?

**Comment by justMaku** [comment]
I'll merge and update the crest of the code mysel, thanks a lot for the help!

**Comment by finngaida** [comment]
Sure thing

**Pull request merged**